runtime/drivers/sqlite/registry.go

```diff
@@ -30,21 +31,33 @@ func (c *connection) findInstances(_ context.Context, whereClause string, args .
 	// Override ctx because sqlite sometimes segfaults on context cancellation
 	ctx := context.Background()
 
-	sql := fmt.Sprintf("SELECT id, olap_driver, olap_dsn, repo_driver, repo_dsn, embed_catalog, created_on, updated_on FROM instances %s ORDER BY id", whereClause)
+	sql := fmt.Sprintf("SELECT id, olap_driver, olap_dsn, repo_driver, repo_dsn, embed_catalog, created_on, updated_on, env, project_env FROM instances %s ORDER BY id", whereClause)
 
 	rows, err := c.db.QueryxContext(ctx, sql, args...)
 	if err != nil {
 		return nil, err
 	}
 	defer rows.Close()
 
+	// sqlite doesn't support maps need to read as bytes and convert to map
+	var env, projectEnv []byte
```

Comment:
Move this declaration inside the loop. In case DB returns `nil`, it might do nothing instead of assign `nil` to the pointer, causing us to re-use env vars from previous loop iteration

Reply:
Should not be the case I think but no harm in declaring it inside loop as well.